packages/hooks/src/useCFDAccounts.ts

```diff
@@ -0,0 +1,18 @@
+import { useStore } from '@deriv/stores';
+
+const useCFDAccounts = () => {
```

Comment:
Help to add some documentation for this.
Soon we will add new platforms such as cTrader, DerivEz and SwapFree and it is good to have more clarity!

Reply:
done :heavy_check_mark: